includes/model/class-follower.php

```diff
@@ -163,15 +170,17 @@ public function save() {
 		}
 
 		$args = array(
-			'ID'           => $this->get__id(),
-			'guid'         => esc_url_raw( $this->get_id() ),
-			'post_title'   => wp_strip_all_tags( sanitize_text_field( $this->get_name() ) ),
-			'post_author'  => 0,
-			'post_type'    => Followers::POST_TYPE,
-			'post_name'    => esc_url_raw( $this->get_id() ),
-			'post_excerpt' => sanitize_text_field( wp_kses( $this->get_summary(), 'user_description' ) ),
-			'post_status'  => 'publish',
-			'meta_input'   => $this->get_post_meta_input(),
+			'ID'                    => $this->get__id(),
+			'guid'                  => esc_url_raw( $this->get_id() ),
+			'post_title'            => wp_strip_all_tags( sanitize_text_field( $this->get_name() ) ),
+			'post_author'           => 0,
+			'post_type'             => Followers::POST_TYPE,
+			'post_name'             => esc_url_raw( $this->get_id() ),
+			'post_excerpt'          => sanitize_text_field( wp_kses( $this->get_summary(), 'user_description' ) ),
+			'post_status'           => 'publish',
+			'post_content'          => $this->to_json(),
```

Comment:
We were not sure if the default filters used on the content would interfere with the JSON. Do you have some experience with using JSON as post-content?

Reply:
Default filters and also other plugins should not mess with post_types they don't know IMHO (in an ideal world) but it would be a case, where I would reach out to the community (via Mastodon) and see if I can gather some opinions.

Reply:
Another point: the json representation of the follower actor which is saved in the `post_content` and the according inbox used by WordPress which is saved in the `post_content_filtered` are recoverable things. They are just cached. They can be retrieved at any time, as long as the actor id which is stored in the `post_guid` remains intact.

Reply:
I also thought about adding the inbox to the `to_ping` field instead!?!

Reply:
The `to_ping` field is used heavily in the core. And to what I found in the internet posted by Automattic devs almost two decades ago, it seems like our transformations and extractions of shared inboxes are perfect use cases of `post_content_filtered`.